hooks/generateRandomColor.js

```diff
@@ -0,0 +1,19 @@
+import {useState} from 'react';
+
+    /**
+   * @param {number} min
+   * @param {number} max
+   */
+    function randomInt(min, max) {
+      return Math.floor(Math.random() * (max - min) ) + min;
```

Comment:
Good call out that this color will be randomly generated on every refresh. The easiest solution would be passing in a random seed based on the post’s ID so the color generated for a particular post is the same every time.

Reply:
We could also just store the color in hex itself.

Reply:
We can, but that doesn't solve ensuring each post has a unique color. At least for now the color will only be used to identify the post on the map.